keyboards/gray_studio/cod67/rules.mk

```diff
@@ -11,19 +11,23 @@ MCU = atmega32u4
 #   ATmega328P   USBasp
 BOOTLOADER = atmel-dfu # actually lufa-ms
 
+# Mass storage bootloader on the COD67 uses bin files
+FIRMWARE_FORMAT = bin
+
 # Build Options
 #   change yes to no to disable
 #
-BOOTMAGIC_ENABLE = no      # Virtual DIP switch configuration(+1000)
-MOUSEKEY_ENABLE = yes       # Mouse keys(+4700)
+BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE = no        # Mouse keys(+4700)
```

Comment:
did you disable this because size issues? As it takes away functionality.

Reply:
Yep, when I got close to the firmware size limit then enabling the console would cause the keyboard not to start the firmware. I'll put that back, well spotted.